libdevcore/Common.h

```diff
@@ -284,7 +289,11 @@ class Timer
 #define DEV_TIMED_FUNCTION DEV_TIMED_SCOPE(__PRETTY_FUNCTION__)
 #endif
 
+#ifdef __MIC__
+#define DEV_TIMED_ABOVE(S, MS) for (::std::pair<dev::TimerHelper, bool> __eth_t(::dev::TimerHelper(S, MS), true); __eth_t.second; __eth_t.second = false)
```

Comment:
This is actually part of c++11: http://en.cppreference.com/w/cpp/language/operator_alternative

If it is easy to make it compatible for non-compliant compilers, please just introduce spaces.